run_combination_test.py

```diff
@@ -71,6 +71,7 @@ def get_shuffle_params(params, index):
     if params['h5py'] == '2.6':
         conf['requires'].append('h5py<2.7')
     elif params['h5py'] == '2.5':
+        conf['requires'].append('numpy<1.10')
```

Comment:
And do you mean this line installs the latest numpy? "numpy<1.11" doesn't work correctly?